config/locales/en/helpers.yml

```diff
@@ -64,7 +66,9 @@ en:
       steps_address_lookup_form:
         postcode: This must be a valid UK postcode. For example, SW1A 2AA.
       steps_case_urn_form:
-        urn: For example, ‘12 AB 3456789’.
+        urn: For example, ‘12 AB 3456789’
```

Comment:
Is there a reason the . was removed?  Looks like it's still in figma

Reply:
Oh I think that was by mistake. Will re-add it.

Reply:
Hmm, now looking at it, perhaps we need to clarify the position and either remove all full stop from all the hint texts in the service, or add them where we are missing them 🤣 
Because we have a mix of style I believe. In the design system guide, they sometimes add them, sometimes don't, so it is a bit unclear!